btr-web/btr-common-components/middleware/setupAuth.global.ts

```diff
@@ -8,6 +8,25 @@ export default defineNuxtRouteMiddleware(async (to) => {
       { url: kcURL, realm: kcRealm, clientId: kcClient },
       to.params.currentAccountId as string || to.query.currentAccountId as string
     )
+
+    if (process.client && sessionStorage?.getItem('FAKE_LOGIN')) {
```

Comment:
Can this be problematic in prod env ?

Reply:
haha well in dev/test/prod if they set this it would just cause them to have the fake data in their header and 0 access to the apis